### DISCUSSION TITLE
Would this work on a sever? if so, how?

### DISCUSSION THREAD

**Discussion by BenR001** [Joshua-Ashton/VPhysics-Jolt#147] [Q&A]
See the title for my question.

**Comment by FireGrey** [comment]
Yes, just follow the installation instructions and put it in your server's bin directory.
Just need to make sure you're using 64bit for your server.

**Comment by Mika32150** [comment]
It doesn't work for me... 
https://gyazo.com/146243ebb424fae04977cad67ef7c799
